### PR TITLE
Dashing devel changes during moveit2 Windows bringup

### PR DESCRIPTION
**Pull request by lilustga**
Removed Assimp dependency export.
Added symbol export for symbol visibility on Windows.
Added math defines.

### CODE REVIEW

File: include/geometric_shapes/bodies.h [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #ifndef GEOMETRIC_SHAPES_BODIES_
 #define GEOMETRIC_SHAPES_BODIES_
 
-#if __cplusplus <= 199711L
+#if __cplusplus <= 199711L && ! defined (WIN32)
```
Comment (ooeygui) [inline include/geometric_shapes/bodies.h:40]: Why is this needed if you set __cplusplus?

**Review by ooeygui — APPROVED** [review]

File: include/geometric_shapes/bodies.h (hunk 1 above)
Comment (lilustga) [reply include/geometric_shapes/bodies.h:40]: because it was failing in moveit2 wherever the header was included. So _cplusplus flag would have been needed in all the CMake files
